test/bloomFilterReader.test.ts

```diff
@@ -0,0 +1,48 @@
+import {expect} from "chai"
+import { parseBloomFilterOffsets } from '../lib/bloomFilterReader';
+import { ColumnChunk, ColumnChunkData, ColumnData } from "../lib/types/types.js";
+// const bloomFilterReader = require("../lib/bloomFilterReader.js");
```

Comment:
Extra comment. This one threw me for a second, "A test of the bloomFilterReader that doesn't use the bloomFilterReader?" Then I saw the import.